### PR TITLE
[Refactor] Ease rewriter import for dev-1.x

### PR DESCRIPTION
**Pull request by grimoire**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

https://github.com/open-mmlab/mmdeploy/pull/1166

## Modification

mmdet3d and mmrotate has been skipped since they has not been supported yet.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by AllentDan** [comment]
May resolve conflicts.

**Review by AllentDan — APPROVED** [review]
LGTM after conflicts resolved

File: mmdeploy/codebase/mmcls/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from . import models  # noqa: F401,F403
 from .deploy import *  # noqa: F401,F403
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmcls/__init__.py:3]: should update `mmdeploy/codebase/mmaction/models/recognizers/__init__.py` as well

File: mmdeploy/codebase/mmocr/models/text_recognition/__init__.py [hunk 2]
```diff
@@ -1,14 +1,8 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 # from .base import base_recognizer__forward
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmocr/models/text_recognition/__init__.py:2]: may remove this line.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**